### PR TITLE
dllib keras like model summary support returning string

### PR DESCRIPTION
**Pull request by dding3**
Fix https://github.com/intel-analytics/BigDL/issues/4318

### CODE REVIEW

**Comment by dding3** [comment]
jekins pass. http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-Dllib-Validation/398/

File: python/dllib/src/bigdl/dllib/keras/engine/topology.py [hunk 1]
```diff
@@ -500,10 +500,10 @@ def summary(self, line_length=120, positions=[.33, .55, .67, 1.]):
                    If the field has a larger length, the remaining part will be trimmed.
                    If the field has a smaller length, the remaining part will be white spaces.
         """
-        callZooFunc(self.bigdl_type, "zooKerasNetSummary",
-                    self.value,
-                    line_length,
-                    [float(p) for p in positions])
+        return callZooFunc(self.bigdl_type, "zooKerasNetSummary",
```
Comment (jason-dai) [inline python/dllib/src/bigdl/dllib/keras/engine/topology.py:503]: I think a more reasonable way to deal with it is:
1) Calling `summary` in Scala should print the message in Scala
2) Calling `summary` in Python should print the message in Python (no print in Scala)
3) `summary` can optionally return a string

**Comment by yangw1234** [comment]
It is ready to update on the user group thread requesting this feature?

**Comment by dding3** [comment]
I am thinking waiting one more day for the nightly build whl?

**Pull request merged**